pkg/cli/admin/prune/imageprune/helper.go

```diff
@@ -40,6 +40,10 @@ func (ba isByAge) Less(i, j int) bool {
 	return ba[i].CreationTimestamp.After(ba[j].CreationTimestamp.Time)
 }
 
+// ErrNoImageFound is an error returned when we could not find an image from where we can
+// derive the image registry host.
+var ErrNoImageFound = fmt.Errorf("no image found from where we can derive registry host")
```

Comment:
nit - perhaps rename to `ErrRegistryHostNotFound`:

```suggestion
// ErrRegistryHostNotFound is an error returned when we could not find an image from where we can
// derive the image registry host.
var ErrRegistryHostNotFound = fmt.Errorf("no image found from where we can derive registry host")
```